firefox-ios/Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -93,7 +93,7 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
         label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.HomeTabBanner.descriptionLabel3
     }
 
-    private lazy var goToSettingsButton: PrimaryRoundedButton = .build { button in
+    private lazy var goToSettingsButton: PrimaryRoundedGlassButton = .build { button in
```

Comment:
Should the other cards maybe use the `PrimaryRoundedGlassButton` as well? And should the secondary button have glass as well to be consistent?